.circleci/config.yml

```diff
@@ -62,105 +26,25 @@ commands:
             - "~/.m"
```

Comment:
`~/.m`? Was this meant to be `~/.npm` originally? Or `~/.m2`? We're no longer building with node in these tests, so not needed anyway?

Reply:
@srikanthops Can you dig into this?

Reply:
it should ~/.m2 (maven local repository)
I will remove it as it is not relevant  here..